lib/ts/recipe/passwordless/components/features/signInAndUp/index.tsx

```diff
@@ -264,81 +267,122 @@ export const SignInUpFeature: React.FC<
 export default SignInUpFeature;
 
 function getModifiedRecipeImplementation(
-    originalImpl: RecipeInterface,
+    recipe: Recipe,
     dispatch: React.Dispatch<PasswordlessSignInUpAction>,
     callingConsumeCodeRef: React.MutableRefObject<boolean>
 ): RecipeInterface {
     return {
-        ...originalImpl,
+        ...recipe.recipeImpl,
         createCode: async (input) => {
             let contactInfo;
             if ("email" in input) {
                 contactInfo = input.email;
+
+                const validationRes = await recipe.config.validateEmailAddress(input.email);
```

Comment:
This is also done in the forms where we are using createCode.

Reply:
This validation has also been added to (or modified to throw STGeneralError in):
- `themes/signInUp/emailForm.tsx` (email only)
- `themes/signInUp/emailOrPhoneForm.tsx`
- `themes/signInUp/phoneForm.tsx` (phone only)

Reply:
That's my point: this is done both before and during `createCode`. If it's done during the call it should be removed from those forms I think.

Reply:
@porcellus @nkshah2 this comment is not resolved.

Reply:
@rishabhpoddar This has been solved but just to confirm, @porcellus this is good to be resolved right?